sql/moz-fx-data-shared-prod/firefox_ios/retention/view.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.retention`
+AS
+SELECT
+  *,
+  -- not sure what the definition of lifecycle_stage is, so this is a placeholder
+  -- for now just to show where we could place it.
```

Comment:
Here is the one Katie is using for engagement:

[sql/moz-fx-data-shared-prod/telemetry/desktop_engagement_client/view.sql](https://github.com/mozilla/bigquery-etl/blob/1322bf67820840e03547def793b031cb26c9e269/sql/moz-fx-data-shared-prod/telemetry/desktop_engagement_client/view.sql#L6-L16)

Reply:
thanks, I'm not sure about this part:
``` 
WHEN DATE_DIFF(submission_date, first_seen_date, 
  BETWEEN 1 AND 27
  THEN 'repeat_user'
```

We are already counting "repeat_clients" here: https://github.com/mozilla/bigquery-etl/pull/5304/files#diff-79ae2c99183febe6472eb8275fc03cd44c973d4f883671ac7f3580d95ef0a5e1R59

And the definition for repeat_client upstream is that they're active more than once whereas here we just consider how old is the client. Could this create confusion?